packages/blockchain-api/src/schema.ts

```diff
@@ -61,11 +61,14 @@ export interface CurrencyConversionArgs {
   sourceCurrencyCode?: string
   currencyCode: string
   timestamp?: number
+  knownExchangeRates?: { [pair: string]: string }
```

Comment:
Same as comment below- are multiple exchange rates needed or is this always a single rate?